pages/card/card .html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Card</title>
+</head>
+<link rel="stylesheet" href="./card.css">
+<link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+
+<body>
+    <section class="card-content vertical-card-container">
+        <div class="card">
+            <img class="card-img" src="https://picsum.photos/250/300" alt="" />
+            <div class="card-info">
+                <div class="card-title">
+                    <div>
+                        <h3>Lorem ipsum</h3>
+                        <p class="card-description">Lorem ipsum dolor sit.</p>
+                    </div>
+                    <i class="fas fa-heart" aria-hidden="true"></i>
```

Comment:
This logo won't be visible to screen readers, also add `title` attribute to your icons.